src/Promo/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './style.scss';
+import btnBlack from '../image/button-black.png';
+import btnWhite from '../image/Button White.png';
```

Comment:
buttons using img? Pls do it using your hands

Reply:
done